docs/admin/config/index.mdx

```diff
@@ -33,3 +33,4 @@ This page documents how to configure a Sourcegraph instance. For deployment conf
 - [Loading configuration via the file system](/admin/config/advanced_config_file)
 - [Restore postgres database from snapshot](/admin/config/restore/)
 - [Enabling database encryption for sensitive data](/admin/config/encryption)
+- [Configuring Sourcegraph in private networks](/admin/config/private_network)
```

Comment:
thanks for clarifying, I wasn't sure what the best practice was considering both are used throughout our docs.

Reply:
Yeah no worries. I am trying to make the link paths consistent. Otherwise, I'll have to add redirects. 🙈 

Please merge the PR if you've the desired approvals.